unstructured/partition/common.py

```diff
@@ -88,8 +89,9 @@ def normalize_layout_element(
     # in order to add coordinates metadata to the element.
     coordinates = layout_dict.get("coordinates")
     element_type = layout_dict.get("type")
-    if layout_dict.get("prob"):
-        class_prob_metadata = ElementMetadata(detection_class_prob=float(layout_dict.get("prob")))
+    prob = layout_dict.get("prob")
+    if prob and isinstance(prob, (int, str, float)):
```

Comment:
I'm actually not sure if we're seeing cases of that var type, but float can take them if so! This line was just added because I was getting typing errors further down telling me that `prob` var could be `None` when we go to pass it to the `float()` call.